### ISSUE TITLE
Add Warning if Bags Don't Have Enough Room for Total Outfit Withdrawal from Bank or Swap

### ISSUE THREAD

**Issue by TheDreadGazeebo** [cdmichaelb/Outfitter#56]
When I am trying to switch to a banked outfit, the addon attempts to withdraw the entire item set from the bank all at once. this is fine unless you have other items in your inventory. It will remove all the items i am currently wearing and attempt to deposit them all to the bank, and then attempt to withdraw the other set and equip it. if your bank/inventory are both close to full, it will remove and bank all the items it can, and then get stuck when it runs out of inventory space. This leaves my character half-naked and my item sets in complete disarray. Now I have to rebuild all my sets manually because all items are left with a "question mark". This basically makes the addon useless unless I keep those inventory slots open all the time. It seems like this issue could be easily fixed by unequipping/banking/equipping the items one at a time instead of attempting to remove the entire set and then getting stuck when it runs out of space. Maybe a toggleable option?

**Comment by Gogo1951** [comment]
Yeah... but if we add a "can't use this add-on because you don't 18 slots in your bags free" that creates some other issues. 

Going to look into adding an error. Thanks!
